plugins/BEdita/Core/src/Model/Table/ObjectsTable.php

```diff
@@ -73,58 +74,51 @@ public function initialize(array $config)
         $this->setDisplayField('title');
 
         $this->addBehavior('Timestamp');
-
         $this->addBehavior('BEdita/Core.DataCleanup');
-
         $this->addBehavior('BEdita/Core.UserModified');
-
         $this->addBehavior('BEdita/Core.Relations');
-
         $this->addBehavior('BEdita/Core.CustomProperties');
+        $this->addBehavior('BEdita/Core.UniqueName');
+        $this->addBehavior('BEdita/Core.Searchable', [
+            'fields' => [
+                'title' => 10,
+                'description' => 7,
+                'body' => 5,
+            ],
+        ]);
 
         $this->belongsTo('ObjectTypes', [
             'foreignKey' => 'object_type_id',
             'joinType' => 'INNER',
             'className' => 'BEdita/Core.ObjectTypes'
         ]);
-
         $this->hasMany('DateRanges', [
             'foreignKey' => 'object_id',
             'className' => 'BEdita/Core.DateRanges',
             'saveStrategy' => 'replace',
         ]);
-
-        $this->belongsTo('CreatedByUser', [
+        $this->belongsTo('CreatedByUsers', [
```

Comment:
Why the name change? A bette naming convention?

Reply:
Association names should be plural, according to CakePHP's conventions. Since the refactor didn't involve many files, I took the chance. 😊